src/test/java/com/iexec/worker/compute/post/web2/UploaderServiceTests.java

```diff
@@ -128,27 +129,26 @@ void shouldNotUploadToDropboxSinceWrongAccount() throws DbxException, IOExceptio
     }
     //endregion
 
-    //region createRestTemplate
-    @Test
-    void shouldCreateRestTemplate() {
-        assertNotNull(uploaderService.createRestTemplate());
-    }
-    //endregion
-
     //region uploadToIpfsWithIexecProxy
     @Test
     void shouldUploadToIpfsWithIexecProxy() throws IOException {
         final String baseUrl = "baseUrl";
```

Comment:
Minor suggested improvement, not related to the initial PR, but it could be time to use true URLs here (there are several places where this could be done)
```suggestion
        final String baseUrl = "http://localhost";
```